### DISCUSSION TITLE
Unresolved class 'fragulanavhostfragment'

### DISCUSSION THREAD

**Discussion by peakandroid** [massivemadness/Fragula#5] [Q&A]
I am using Fragment navigation. I followed the steps as mentioned but in the xml, when I replace NavHostFragment with FragulaNavHostFragment, I get this error - Unresolved class 'fragulanavhostfragment'.

**Comment by massivemadness** [comment]
Can you show me your xml with navigation container? Sounds like you wrote the wrong path to a class, it should be:
```xml
android:name="com.fragula2.FragulaNavHostFragment" 
```

**Reply by peakandroid** [reply]
Hi, Thank you for response. 

Below is the xml file - 
[xmlversion.docx](https://github.com/massivemadness/Fragula/files/9671129/xmlversion.docx)

**Reply by peakandroid** [reply]
Errors after building and trying to run - 

![Screen Shot 2022-09-29 at 9 08 52 AM](https://user-images.githubusercontent.com/112940011/192944041-c8f1ede2-a99f-42fa-85f3-d759b2c114e7.png)

**Reply by massivemadness** [reply]
Looks like Gradle cannot download the dependency 🤔
Do you have `mavenCentral()` repo in your top-level `build.gradle`?

**Reply by peakandroid** [reply]
Yes, top level build.gradle is 


buildscript{
    ext.kotlin_version = '1.7.0-RC'
    repositories {
        mavenCentral()
        google()
        jcenter()
        flatDir {
            dirs 'libs'
        }// here
    }
    dependencies {
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
        def nav_version = "2.5.1"
        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$nav_version"
    }
}

plugins {
    id 'com.android.application' version '7.2.2' apply false
    id 'com.android.library' version '7.2.2' apply false
    id 'org.jetbrains.kotlin.android' version '1.6.10' apply false

}
apply from: "config.gradle"

task clean(type: Delete) {
    delete rootProject.buildDir
}

**Reply by massivemadness** [reply]
Can't see the `allprojects { ... }` block, is it placed in config.gradle?

**Reply by massivemadness** [reply]
Try this:
```groovy
allprojects {
    repositories {
        google()
        mavenCentral()
    }
}
```
